frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -53,18 +53,6 @@ export const trendsLogic = kea<trendsLogicType>([
     })),
 
     reducers(({ props }) => ({
-        toggledLifecycles: [
-            ['new', 'resurrecting', 'returning', 'dormant'],
-            {
-                toggleLifecycle: (state, { lifecycleName }) => {
-                    if (state.includes(lifecycleName)) {
-                        return state.filter((lifecycles) => lifecycles !== lifecycleName)
-                    }
-                    return [...state, lifecycleName]
-                },
-                setLifecycles: (_, { lifecycles }) => lifecycles,
```

Comment:
Should this `setLifecycles` still set `toggledLifecycles` like it used to? E.g. by calling setFilters or so? I think it's still being called.

Reply:
![data-lifecycle](https://user-images.githubusercontent.com/984817/222831892-2148623a-0626-4590-9b91-ff28a3cc0188.gif)